src/Client/pages/WishList.fs

```diff
@@ -81,7 +82,7 @@ let update (msg:WishListMsg) model : Model*Cmd<WishListMsg> =
     | AddBook ->
         if Validation.verifyBook model.NewBook then
             let wishList = { model.WishList with Books = (model.NewBook :: model.WishList.Books) |> List.sortBy (fun b -> b.Title) }
-            { model with WishList = wishList; NewBook = Book.empty }, postWishListCmd(model.Token,wishList)
+            { model with WishList = wishList }, postWishListCmd(model.Token,wishList)
```

Comment:
OK, but if ``NewBook = Book.empty`` is used (which seems fair enough) then the input fields aren't cleared in the UI (maybe that's the bug).  Well, the authors field is cleared but no others.  This leads dangling text in the fields, and if you press "add book" again it says no text has been addded, see picture.

![image](https://cloud.githubusercontent.com/assets/7204669/23221513/3563f626-f91d-11e6-8939-a6e43548f41b.png)